x-pack/metricbeat/module/sql/query/query.go

```diff
@@ -49,7 +49,8 @@ type config struct {
 	ResponseFormat string `config:"sql_response_format"`
 	Query          string `config:"sql_query" `
 
-	Queries []query `config:"sql_queries" `
+	Queries      []query `config:"sql_queries" `
+	MergeQueries bool    `config:"request_merge"`
```

Comment:
As commented in https://github.com/elastic/beats/issues/31806#issuecomment-1148568527, I don't like so much the name of `request_merge` for this option, it is not very meaningful. What are you merging? Is this a request that may not be fulfiled?

I would suggest something like `merge_results`, `merge_events` or `single_event`.
```suggestion
	MergeResults bool    `config:"merge_results"`
```
```suggestion
	SingleEvent bool    `config:"single_event"`
```

Reply:
Changed the flag name as merge_results and also the variable. **merge_results** sounds fine with me, so let's go ahead with it.